lib/code_fund_web/controllers/page_controller.ex

```diff
@@ -16,11 +16,35 @@ defmodule CodeFundWeb.PageController do
     render(conn, "index.html", stats: stats)
   end
 
+  def advertiser(conn, _params) do
+    render(conn, "advertiser.html")
+  end
+
+  def publisher(conn, _params) do
+    render(conn, "publisher.html")
+  end
+
   def deliver_contact_form(conn, params) do
     Emails.contact_form_email(params) |> Mailer.deliver_now()
 
     conn
     |> put_flash(:info, "Your request was submitted successfully")
     |> redirect(to: page_path(conn, :index))
   end
+
+  def deliver_advertiser_form(conn, %{"form" => form}) do
+    Emails.advertiser_form_email(form) |> Mailer.deliver_now()
```

Comment:
Feels like this and the publisher form could be one function, pass in the `:publisher` or `:advertiser` as an atom, `apply` the corresponding function from `Emails`.